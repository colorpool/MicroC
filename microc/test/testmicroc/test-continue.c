void main(){
    int i;
    i=0;
    do{
        i=i+1;
        if (i%2==0) 
            continue;
        print 'd' : i;
    }
    while(i<100);

}