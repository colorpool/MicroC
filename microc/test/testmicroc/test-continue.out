24 19 0 5 25 15 1 13 0 0 1 0 0 12 15 -1 13 0 0 1 13 0 0 1 11 0 1 1 12 15 -1 13 0 0 1 11 0 2 5 0 0 6 17 48 16 16 16 50 15 0 13 0 0 1 11 22 15 -1 15 0 13 0 0 1 11 0 100 7 18 16 15 -1 21 -1